tests/shared/src/test/scala/scala/meta/tests/parsers/LitSuite.scala

```diff
@@ -5,93 +5,104 @@ import scala.meta._
 import scala.meta.dialects.Scala211
 
 class LitSuite extends ParseSuite {
+
+  // Structure does not always return consisten results
+  private def assertLiteral(obtained: Term, expected: Lit) {
```

Comment:
I am not really sure what's happening here. why is regular assertTree not working on 
```
assertLiteral(term("0xCAFEBABE"), Lit.Int(-889275714))
assertLiteral(term("42.42"), Lit.Double(42.42))
assertLiteral(term("42.42f"), Lit.Double(42.42f))
```